developers/README.md

```diff
@@ -3,34 +3,28 @@
 ## Table of Contents
 
 * [Development Environment](#development-environment)
-  * [Local Environment](#local-environment)
-  * [Virtual Machine](#virtual-machine)
+  * [Virtual Machines](#virtual-machines)
   * [Docker](#docker)
 * [Contributing](#contributing)
 * [IRC](#irc)
 
 ## Development Environment
 
-Before hacking on BonnyCI, we'll need to set up a development environment. There are three basic ways to do so:
+Before hacking on BonnyCI, we'll need to set up a development environment. There are two basic ways to deploy BonnyCI on your local machine:
 
-  1. [On our local machine](#local-environment)
-  2. [In a virtual machine](#virtual-machine)
-  3. [In a Docker container](#docker)
+  1. [In virtual machines](#virtual-machines)
+  2. [In Docker containers](#docker)
 
-### Local Environment
+### Virtual Machines
 
 Supported development platforms:
```

Comment:
Since this will be moved above, we can remove this. The `Virtual Machines` section should be reserved for [running hoist in a local VM environment](https://github.com/eggshell/hoist/blob/master/README.rst). The `Docker` section should be reserved for running hoist in a Docker environment.

My $0.02: The instructions for setting up a dev environment should be a one-stop, rather than having essentially the same document with extremely minor changes for VMs vs Docker. Once Docker support is added to hoist, we can make it apparent in the `dev-environment/*.md` docs that you don't need to install vagrant/virtualbox if you plan on using docker (and vice versa).